overview.md

```diff
@@ -4,81 +4,86 @@ css_id: overview
-These systems typically update the applications installed on client systems to
-introduce new features, enhancements, and security fixes.
+### Purpose, or Why Get TUF? ###
 
-Three major classes of software update systems are:
+There are literally thousands of different software update systems in common
+use today. (In fact the average Windows user has about [two
+dozen](http://secunia.com/gfx/pdf/Secunia_RSA_Software_Portfolio_Security_Exposure.pdf)
+different software updaters on their machine!)
 
-* **Application updaters** which are used by applications to update
-   themselves. For example, Firefox updates itself through its own application
-   updater.
+What these very different systems have in common is that they all identify,
+locate, and download updates for software that can add new functionalities or
+address old vulnerabilities. Software is rarely ever static, and some repositories
+receive updates on software or project metadata [every few minutes](https://ssl.engineering.nyu.edu/papers/kuppusamy_nsdi_16.pdf). This
+growing flow of updates has also created a need for better
+ways to protect the systems that manage them. Though a number of strategies have
+been introduced and used over the last decade or so to enhance the
+authenticity—and by extension, the security of update systems—most have drawbacks
+that have left update files and the repositories that host them.
+vulnerable to a number of attacks.
 
-* **Library package managers** such as those offered by many
-   programming languages for installing additional libraries. These are systems
-   such as Python's pip/easy_install + PyPI, Perl's CPAN, Ruby's RubyGems, and
-   PHP's Composer.
+TUF was launched almost a decade ago as a way to build system resilience against
+key compromises and other attacks that can spread malware or compromise a repository.
+The primary goals behind its design are:
 
-* **System package managers** used by operating systems to update and
-   install all of the software on a client system. Debian's APT, Red Hat's YUM,
-   and openSUSE's YaST are examples of these.
+* to provide a framework (a set of libraries, file formats, and utilities)
+that can be used to secure new and existing software update systems.
 
-### Our approach ###
+* to provide the means to minimize the impact of key compromise
 
-There are literally thousands of different software update systems in common
-use today. (In fact the average Windows user has about [two
-dozen](http://secunia.com/gfx/pdf/Secunia_RSA_Software_Portfolio_Security_Exposure.pdf)
-different software updaters on their machine!)
+* to be flexible enough to meet the needs of a wide variety of software update systems.
 
-We are building a library that can be universally (and in most cases
-transparently) used to secure software update systems.
+* to be easy to integrate with existing software update systems.
 
-### Overview ###
+### Software Updates 101###
```

Comment:
I think you need a space between the text and `###`, otherwise it shows up as:

```
Software Updates 101###
```

Reply:
Yup. Fixed.